exercism/annalyns-infiltration/LincolnInitialSolution.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+
+static class QuestLogic
+{
+    public static bool CanFastAttack(bool knightIsAwake)
+    {
+        if(knightIsAwake){
+            return false;
+        }
+        else{
+            return true;
+        }
+    }
+
+    public static bool CanSpy(bool knightIsAwake, bool archerIsAwake, bool prisonerIsAwake)
+    {
+        if(knightIsAwake || archerIsAwake || prisonerIsAwake){
+            return true;
+        }
+        else{
+            return false;
+        }
+    }
+
+    public static bool CanSignalPrisoner(bool archerIsAwake, bool prisonerIsAwake)
+    {
+        if (archerIsAwake){
+            return false;
+        }
+        else{
+            if(prisonerIsAwake){
```

Comment:
There is an `else if` [statement](https://www.w3schools.com/cs/cs_conditions_elseif.php).